src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -176,6 +190,9 @@ class NexusPublishPlugin : Plugin<Project> {
         val mavenPublications = project.the<PublishingExtension>().publications.withType<MavenPublication>()
         mavenPublications.configureEach {
             val publication = this
+            initializeTask {
+                dependsOn(getStagingProfileTask)
+            }
```

Comment:
In practice, this dependency should be automatically set by Gradle when you configure `stagingProfileId.set(getStagingProfileTask.map...`

I guess it does not hurt, however, it is not needed either.

Reply:
Good point. We don't actually want that dependency when `stagingProfileId` is set explicitly. Thus, we should remove it.